packages/plugin-infra/src/web3/utils/mergeNFTList.ts

```diff
@@ -0,0 +1,14 @@
+import type { ERC721TokenDetailed } from '../../web3-token-types'
+import { uniqWith } from 'lodash-unified'
+import { isSameAddress } from './isSameAddress'
+
+export function mergeNFTList(NFTList: ERC721TokenDetailed[]) {
+    return uniqWith(
+        NFTList,
+        (a, b) =>
+            isSameAddress(a.contractDetailed.address, b.contractDetailed.address) &&
```

Comment:
Please implement this method for every chain (define `Utils.mergeNFTList`) instead of a public one in case of a chain not suite for the current implementation.

Reply:
done